### PR TITLE
Hide 404 pattern from inserter

### PR DESCRIPTION
**Pull request by jffng**
**Description**

This PR hides the 404 pattern from the block inserter. I'm not actually sure if we want to do this, but thought I'd open a PR to discuss.

**Testing Instructions** 

- Open the pattern inserter and go to the General patterns
- Verify the 404 pattern does not show up

### CODE REVIEW

File: inc/patterns/general-404.php [hunk 1]
```diff
@@ -15,4 +15,5 @@
 					<!-- wp:spacer {"height":128} -->
 					<div style="height:128px" aria-hidden="true" class="wp-block-spacer"></div>
 					<!-- /wp:spacer -->',
+	'inserter'   => false,
```
Comment (kjellr) [inline inc/patterns/general-404.php]: Really minor, but can we move this up above the content? I think it's less likely to be missed that way. Also, do we still need the category attribute?

Comment (jffng) [reply inc/patterns/general-404.php]: > Also, do we still need the category attribute?

Nope!

> can we move this up above the content?

👍

**Comment by kjellr** [comment]
This seems ok, but what do you think about creating a new `hidden-` filename prefix for this and for the one that'll likely land in #183? It seems a little weird to name them `general-` and `header-` etc. 

So this one might become just `hidden-404.php`?

**Review by kjellr — APPROVED** [review]
Looks good! Thanks.

**Pull request merged**